docs/content/en/docs/yaml-crd-ref/metric.md

```diff
@@ -3,3 +3,118 @@ title: KeptnMetric
 description: Define all workloads and checks associated with an application
 weight: 50
 ---
+
+A `KeptnMetric` represents a metric that is collected from a provider.
+Providing the metrics as a CRD into a Kubernetes cluster
+facilitates the reusability of this data across multiple components
+and allows using multiple observability platforms
+for different metrics at the same time.
+
+A `KeptnMetric` looks like the following:
+
+## Yaml Synopsis
+
+```yaml
+apiVersion: metrics.keptn.sh/v?alpha?
+kind: KeptnMetric
+metadata:
+  name: <metric-name>
+  namespace: <application-namespace>
+spec:
+  provider:
+    name: "<named-provider>"
+  query: "<query>"
+  fetchIntervalSeconds: <#-seconds>
+```
+
+## Fields
+
+* **apiVersion** -- API version being used.
+`
+* **kind** -- Resource type.
+   Must be set to `KeptnTaskDefinition`
+
+* **metadata**
+  * **name** -- Unique name of this metric.
```

Comment:
for the naming restrictions, you can refer to the [Kubernetes Naming spec](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names) for all of our CRDs

Reply:
Thanks!  Done!